platform/linux-generic/include/odp_packet_io_internal.h

```diff
@@ -357,6 +357,25 @@ int _odp_lso_create_packets(odp_packet_t packet, const odp_packet_lso_opt_t *lso
 void _odp_pktio_allocate_and_send_tx_compl_events(const pktio_entry_t *entry,
 						  const odp_packet_t packets[], int num);
 
+static inline int pktio_packet_to_pool(odp_packet_t *pkt,
+				       odp_packet_hdr_t **pkt_hdr,
+				       odp_pool_t new_pool)
+{
+	if (new_pool == odp_packet_pool(*pkt))
+		return 0;
+
+	odp_packet_t new_pkt = odp_packet_copy(*pkt, new_pool);
+
+	if (new_pkt == ODP_PACKET_INVALID)
```

Comment:
Since this is fast path function, `odp_unlikely()` could be used. `odp_likely()` could be used in the previous if.